hw06-ATM/src/main/java/homework/Runner.java

```diff
@@ -0,0 +1,12 @@
+package homework;
+
+public class Runner {
+    public static void main(String[] args) {
+        Integer[]cellsNominals={1000,2000,5000,500};
+        ATM atm=new ATM(cellsNominals);
+        atm.addMoney(5000,30);
+        atm.addMoney(2000,10);
+        atm.addMoney(500,10);
+        atm.getMoney(125500);
```

Comment:
```suggestion
        Integer[] cellsNominals = {1000, 2000, 5000, 500};
        ATM atm = new ATM(cellsNominals);
        atm.addMoney(5000, 30);
        atm.addMoney(2000, 10);
        atm.addMoney(500, 10);
        atm.getMoney(125500);
```

Как-то плотно у вас все в исходном коде. Обычно больше пробелов бывает.
В IDEA нажмите Ctrl+Alt+L для форматирования.

В других файлах то же самое.